### ISSUE TITLE
How to generated face from embedding

### ISSUE THREAD

**Issue by sandraplato1234** [deepinsight/insightface#2285]
Hi, 
In the paper "ArcFace: Additive Angular Margin Loss for Deep
Face Recognition" is written that " the pre-trained ArcFace
model can generate identity-preserved and visually reasonable
face images only by using the gradient and BN priors."
How it can be done ? It can generate the face from the embeddings even if the face was not in training data ?
Thanks in advance

**Comment by dxcanh** [comment]
@sandraplato1234 Do you have a solution yet?
